src/main/java/ch/njol/skript/events/EvtItem.java

```diff
@@ -74,10 +75,18 @@
 					.examples("on preparing craft of torch:")
 					.since("2.2-Fixes-V10");
 		}
-		Skript.registerEvent("Pick Up", EvtItem.class, PlayerPickupItemEvent.class, "[player] (pick[ ]up|picking up) [[of] %itemtypes%]")
+		if (hasEntityPickupItemEvent) {
+			Skript.registerEvent("Pick Up", EvtItem.class, EntityPickupItemEvent.class, "[(player|entity)] (pick[ ]up|picking up) [[of] %itemtypes%]")
```

Comment:
Breaks existing scripts.

Reply:
how? 
Its the exact same syntax, with an option for entity. 
If the event doesn't exist, it defaults to the previous (player) event.

Reply:
Shouldn't be difficult to understand how lol, you are changing the event but the syntax is the same.

Reply:
Ah I can see the issue. Ill change it up to include both events.

Reply:
Probably it would be the best to keep the syntax the same (as `[(player|entity)]`) but use EntityPickupItemEvent if used **`entity`**.

Reply:
Okay I made the changes, the entity event is now only fired if `entity` is present